### ISSUE TITLE
1.0.0-pre.51 seems to break something

### ISSUE THREAD

**Issue by DMCTowns** [whitecolor/ts-node-dev#167]
Since updating from 50 to 51, I get the following error when running my development script
```
0 info it worked if it ends with ok                                                                                                                                         
1 verbose cli [ '/usr/local/bin/node',                                                                                                                                      
1 verbose cli   '/usr/local/bin/npm',                                                                                                                                       
1 verbose cli   'run',                                                                                                                                                      
1 verbose cli   'development' ]                                                                                                                                             
2 info using npm@6.14.6                                                                                                                                                     
3 info using node@v10.16.0                                                                                                                                                  
4 verbose run-script [ 'predevelopment', 'development', 'postdevelopment' ]                                                                                                 
5 info lifecycle SenateAuthServiceAPI@0.1.0~predevelopment: SenateAuthServiceAPI@0.1.0                                                                                      
6 info lifecycle SenateAuthServiceAPI@0.1.0~development: SenateAuthServiceAPI@0.1.0                                                                                         
7 verbose lifecycle SenateAuthServiceAPI@0.1.0~development: unsafe-perm in lifecycle true                                                                                   
8 verbose lifecycle SenateAuthServiceAPI@0.1.0~development: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/diccon/Web Server/node/sen$
9 verbose lifecycle SenateAuthServiceAPI@0.1.0~development: CWD: /Users/diccon/Web Server/node/senate/authorization/api                                                     
10 silly lifecycle SenateAuthServiceAPI@0.1.0~development: Args: [ '-c', 'ts-node-dev --respawn --transpileOnly ./src/app.ts' ]                                             
11 silly lifecycle SenateAuthServiceAPI@0.1.0~development: Returned: code: 1  signal: null                                                                                  
12 info lifecycle SenateAuthServiceAPI@0.1.0~development: Failed to exec development script                                                                                 
13 verbose stack Error: SenateAuthServiceAPI@0.1.0 development: `ts-node-dev --respawn --transpileOnly ./src/app.ts`                                                        
13 verbose stack Exit status 1                                                                                                                                              
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)                                               
13 verbose stack     at EventEmitter.emit (events.js:198:13)                                                                                                                
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)                                            
13 verbose stack     at ChildProcess.emit (events.js:198:13)                                                                                                                
13 verbose stack     at maybeClose (internal/child_process.js:982:16)                                                                                                       
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)                                                                               
14 verbose pkgid SenateAuthServiceAPI@0.1.0                                                                                                                                 
15 verbose cwd /Users/diccon/Web Server/node/senate/authorization/api                                                                                                       
16 verbose Darwin 19.5.0                                                                                                                                                    
17 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "development"                                                                                              
18 verbose node v10.16.0                                                                                                                                                    
19 verbose npm  v6.14.6                                                                                                                                                     
20 error code ELIFECYCLE                                                                                                                                                    
21 error errno 1                                                                                                                                                            
22 error SenateAuthServiceAPI@0.1.0 development: `ts-node-dev --respawn --transpileOnly ./src/app.ts`                                                                       
22 error Exit status 1                                                                                                                                                      
23 error Failed at the SenateAuthServiceAPI@0.1.0 development script.                                                                                                       
23 error This is probably not a problem with npm. There is likely additional logging output above.                                                                          
24 verbose exit [ 1, true ]
```

Rolling back to 1.0.0-pre.50 seems to fix the issue

Node version is v12.18.2 (I know it says 10.16.0 in the log, but I've since upgraded and the issue remains).
TS version: 8.10.2

**Comment by whitecolor** [comment]
Change `transpileOnly` -> `transpile-only`

**Comment by Digitum** [comment]
Before version 1.0.0-pre.51, I use this command:
`npx ts-node-dev --respawn --transpileOnly ./src/index.ts --watch`
After version 1.0.0-pre.51 I need to move all arguments to end:
`npx ts-node-dev ./src/index.ts --watch --respawn -transpileOnly`
I have not yet checked if that affects the behaviour and it may very well not be the solution, but can give a hint of where the issue are.

**Comment by dandv** [comment]
@whitecolor: can you take a look at the PR to fix that, #106?

**Comment by Albert-Gao** [comment]
Ah, stuck at this as well

Maybe a more clear error message would be helpful?

![image](https://user-images.githubusercontent.com/18282328/91537504-84151200-e96a-11ea-8c76-34d32105f3a3.png)

Something like output the wrong parameter?
